### PR TITLE
disable the permission requests for hardware ids

### PR DESCRIPTION
**Pull request by dubdabasoduba**

### CODE REVIEW

**Comment by shivenbhatt1** [comment]
@rkodev are there any updates here?

File: gradle.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-VERSION_NAME=2.0.3-SNAPSHOT
+VERSION_NAME=2.0.3-disable_hwid-SNAPSHOT
```
Comment (vincent-karuri) [inline gradle.properties:1]: Could we bump the version?

File: android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonWizardFormActivity.java [hunk 2]
```diff
@@ -8,7 +8,7 @@
 /**
  * Created by keyman on 04/12/2018.
  */
-public class JsonWizardFormActivity extends JsonFormActivity {
+public class JsonWizardFormActivity extends FormConfigurationJsonFormActivity {
```
Comment (vincent-karuri) [inline android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonWizardFormActivity.java:11]: Do we need this change?

Comment (dubdabasoduba) [reply android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonWizardFormActivity.java:11]: Yes, this is to enable form configuration

Comment (vincent-karuri) [reply android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonWizardFormActivity.java:11]: Do we need that for this fix?

Comment (dubdabasoduba) [reply android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonWizardFormActivity.java:11]: Not really, I added it coz I noticed we had missed it during the form config upgrade

Comment (vincent-karuri) [reply android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonWizardFormActivity.java:11]: ok, could we please remove it then?

**Review by vincent-karuri — APPROVED** [review]

**Pull request merged**